pkg/exporter/kafka_proto.go

```diff
@@ -28,6 +28,17 @@ func (kp *KafkaProto) ExportFlows(input <-chan []*flow.Record) {
 	}
 }
 
+func getFlowKey(record *flow.Record) []byte {
+	for k := range record.Id.SrcIp {
```

Comment:
this assumes number of bytes in src and dest are equal .. what if there's an ipv4 and an ipv6 ?

Reply:
maybe check `len` first to return the smallest first (or last) if they aren't equal?
Also, can you add a comment about why we need to sort this? I fear if we look at the code later we might hav forgotten :-)

Reply:
SrcIp and DstIp are not slice but fixed length array:
https://github.com/netobserv/netobserv-ebpf-agent/blob/c62173ac9558581fee03e63955a7eb8941178d05/pkg/ebpf/bpf_bpfel.go#L23

And more general question, is it possible to have a src ipv4 and a dst ipv6? I would have said no but you made me wonder

Reply:
that is probably impossible to have a pkt header "flow" with src and dst from different address family.
/lgtm